### ISSUE TITLE
Running a CoinMachine for anything other than the native token at best shows wrong data, at worst does not work

### ISSUE THREAD

**Issue by area** [JoinColony/colonyDapp#3026]
## Steps to reproduce

1. Deploy CoinMachine
2. Activate CoinMachine, selling a token that is not the colony's native token.
4. Navigate to the `buy-tokens` page

## Expected behaviour

I am able to buy tokens

## Actual behaviour

Client loads up to the 'Loading Buy Tokens' spinner, and 'crashes' to blank white screen. An error is thrown in the console.

```
Uncaught Error: invalid BigNumber string value (arg="value", value="<sub>0</sub>", version=4.0.47)
    throwError errors.js:76
    BigNumber bignumber.js:72
    bigNumberify bignumber.js:182
    totalTokens CoinMachine.tsx:200
    React 3
    CoinMachine CoinMachine.tsx:194
...
```

**Comment by area** [comment]
I don't know what's going on for me here. Maybe best ignore this until I speak to someone to help me pin it down.

**Comment by area** [comment]
Okay, I think I've pinned down what causes this, and I _think_ I've located the cause:

https://github.com/JoinColony/colonyDapp/blob/master/src/data/resolvers/coinMachine.ts#L266-L269

This is getting a token client for the colony's native token, regardless of what the coinmachine is selling.

**Comment by rdig** [comment]
Fixed via 6c9304b8c02054ee274cc4d7acae095cfe0ef6e3 and closed by #3019
